Source/Helpers/YPAdjustableView.swift

```diff
@@ -24,6 +24,14 @@ public class YPAdjustableView: UIView {
         }
     }
 
+    public func adjustViewFramesIfNeeded(cropRect: CGRect, assetSize: CGSize, targetAspectRatio: CGFloat?) {
```

Comment:
nit: should the name of this method just  be `adjustViewFrameIfNeeded` and not the plural name?

Reply:
Renamed.